examples/social-network/websocket-server/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "glee-hello-world",
+  "version": "0.1.0",
+  "description": "Hello world app to improve Glee onboarding ",
```

Comment:
🧐 Should have been websocket-server details right?

Reply:
```suggestion
  "name": "glee-hello-world",
  "version": "0.1.0",
```